portals/edsc/config.json

```diff
@@ -9,7 +9,7 @@
     }
   },
   "footer": {
-    "attributionText": "NASA Official: Stephen Berrick",
+    "attributionText": "",
```

Comment:
I think we should include a value here. Not a real name we'll have to change in the future, but some text. Without something that block of text will never show up in development, which could lead to headaches in the future because we didn't handle something until we noticed it in a production environment.

Reply:
I think we should just default to Doug the tests now don't have those so it shouldn't be confusing. It will just get overwritten with each deployment